cinema/models.py

```diff
@@ -122,8 +122,10 @@ def save(
         )
 
     def __str__(self):
-        return (f"{str(self.movie_session)} "
-                f"(row: {self.row}, seat: {self.seat})")
+        return (
+            f"{str(self.movie_session)} "
+            f"(row: {self.row}, seat: {self.seat})"
+        )
 
     class Meta:
         unique_together = ("movie_session", "row", "seat")
```

Comment:
The 'unique_together' option is deprecated in Django 2.2 and later. It is recommended to use 'constraints' with 'UniqueConstraint' instead. Update the 'unique_together' option to use 'UniqueConstraint' in the Meta class.